src/SFML/Audio/CMakeLists.txt

```diff
@@ -58,6 +58,11 @@ source_group("codecs" FILES ${CODECS_SRC})
 # let CMake know about our additional audio libraries paths (on Windows and OSX)
 if(SFML_OS_WINDOWS)
     set(CMAKE_INCLUDE_PATH ${CMAKE_INCLUDE_PATH} "${PROJECT_SOURCE_DIR}/extlibs/headers/AL")
+    CONFIGURE_FILE(
+        "${SRCROOT}/resource.rc.in"
+        "${SRCROOT}/resource.rc"
```

Comment:
Do not generate files inside the source tree, you can make use of CMAKE_CURRENT_BINARY_DIR
This will also remove the need for your change in .gitignore

Additionally CMake commands are usually written lowercase (both for readability and consistency)

Reply:
K, gimme a bit on that. Also I could've sworn I did do that in lowercase... Huh.

Reply:
Also I realize that you're inserting resource generation in the middle of CMake search path setup. You should keep these configure in a distinct section.

And a lot of things changed in CMake files since you opened the PR, it'd probably be easier to reapply the changes starting from current master.

Reply:
And more thing (sorry it doesn't come all at once :D), as you're repeating more or less the same for each module, you could most likely factorize all the CMake code inside sfml_add_library()

Reply:
Good point on that bit. I'll have a look into it when I get off (today is friday for me).